services/app-web/src/components/SubmissionSummarySection/ContractDetailsSummarySection/ContractDetailsSummarySection.tsx

```diff
@@ -53,6 +55,32 @@ const createCheckboxList = ({
     )
 }
 
+// This function takes a ContractAmendmentInfo and returns two lists of keys sorted by whether they are set true/false
+export function sortAmendedItems(
+    amendmentInfo: ContractAmendmentInfo | undefined
+): [string[], string[]] {
+    const modifiedProvisions = []
+    const unmodifiedProvisions = []
+
+    if (amendmentInfo) {
+        // We type cast this to be the list of keys in the ContractAmendmentInfo
+        const provisions = Object.keys(amendmentInfo) as Array<
+            keyof ContractAmendmentInfo
+        >
+
+        for (const provisionKey of provisions) {
```

Comment:
An abstracted version would be a good re- useable helper to include somewhere with unit tests - maybe in `formHelpers'